packages/vscode-extension/package-lock.json

```diff
@@ -72,19 +72,6 @@
 				"fastq": "^1.6.0"
 			}
 		},
-		"@oracle/suitecloud-cli": {
```

Comment:
I think we have completely removed this dependency. I don't think this should happen.